acceptance-tests/acceptance-pack/src/test/resources/booking_timeout.btm

```diff
@@ -0,0 +1,13 @@
+##############################################################
```

Comment:
Please add the Apache License header for it.

Reply:
@zhfeng  Do worry about that , I will take care of it with a quick patch for it.